### PR TITLE
Closes #3902: truth value of an empty array DeprecationWarning

### LINKED ISSUE #3902: truth value of an empty array DeprecationWarning
**Describe the bug**
When running `make test` @drculhane experienced the following Warning:
```
  /home/drandy/projects/arkouda/arkouda/pdarraycreation.py:492: DeprecationWarning: The truth value of an empty array is ambiguous. Returning False, but in future this will result in an error. Use `array.size > 0` to check that an array is not empty.
    if shape == ():
```

### PR DESCRIPTION
**Pull request by ajpotts**
Mitigates the following pytest warning:
```
arkouda/arkouda/pdarraycreation.py:492: DeprecationWarning: The truth value of an empty array is ambiguous. Returning False, but in future this will result in an error. Use `array.size > 0` to check that an array is not empty.
    if shape == ():
```
Closes #3902: truth value of an empty array DeprecationWarning

### CODE REVIEW

File: arkouda/pdarraycreation.py [hunk 1]
```diff
@@ -489,7 +489,7 @@ def zeros(
     if ndim > get_max_array_rank():
         raise ValueError(f"array rank {ndim} exceeds maximum of {get_max_array_rank()}")
 
-    if shape == ():
+    if isinstance(shape, tuple) and len(shape) == 0:
```
[the remark this answers is not in the document]
Comment (ajpotts) [reply arkouda/pdarraycreation.py:492]: `if not shape` doesn't work b/c cause it has a side effect:
```
In [14]: ak.zeros(0)
Out[14]: array([0.00000000000000000])
```
Which I think is caused by the case when `shape = 0`.

[the remark this answers is not in the document]
Comment (drculhane) [reply arkouda/pdarraycreation.py:492]: Yep.  It seemed like a good idea at the time :) Your fix above is the way to go.

**Review by drculhane — APPROVED** [review]
Looks good.  Thanks.

**Review by jaketrookman — APPROVED** [review]

**Comment by ajpotts** [comment]
I changed this to throw an error when `shape==()`.  We can't yet match numpy in this case:
```
In [2]: x = np.zeros(shape=())

In [3]: x.shape
Out[3]: ()

```

**Review by drculhane — APPROVED** [review]
Looks good.

**Review by jaketrookman — APPROVED** [review]
Looks good

**Pull request merged**